express/src/controllers/manage-account.ts

```diff
@@ -72,9 +72,7 @@ export const showClient = async function (req: Request, res: Response) {
             changeUserAttributes: `/change-user-attributes/${serviceId}/${selfServiceClientId}/${authClientId}?userAttributes=${encodeURI(
                 client.scopes.join(" ")
             )}`,
-            changePublicKey: `/change-public-key/${serviceId}/${selfServiceClientId}/${authClientId}?publicKey=${encodeURI(
-                client.publicKey
-            )}`,
+            changePublicKey: `/change-public-key/${serviceId}/${selfServiceClientId}/${authClientId}?publicKey=${encodeURI(userPublicKey)}`,
```

Comment:
We should be doing more encoding/encrypting (on all the params) to hide plain data but that's a story for another time....